Makefile

```diff
@@ -210,6 +210,10 @@ dev/build:
 dev/fmt:
 	bazel run //hack:update-gofmt
 
+.PHONY: dev/fmt
+dev/golangci-lint:
+	bazel run //hack:golangci-lint
```

Comment:
If we changed this to `bazel run //hack/bin:golangci-lint` do we need the bash script?